packages/@sanity/base/src/components/rovingFocus/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface RovingFocusProps {
+  direction?: 'horizontal' | 'vertical'
+  initialFocus?: 'first' | 'last'
+  loop?: boolean
+  pause?: boolean
+  rootElement: HTMLElement | HTMLDivElement | null
```

Comment:
for my own knowledge: I see some hook apis use refs and I see some other use elements like you have here? i've been wondering which kind of API is better under certain scenarios

Reply:
Not sure actually. For me, it feels easiest to just pass the element directly. But I am not sure if it's the best approach. We use that approach for hooks in `@sanity/ui` such as `useClickOutside` (pass the element, not the ref). :)